### PR TITLE
Update CakeDC Users link to latest branch matching CakePHP 3.1 version

### PR DESCRIPTION
**Pull request by steinkel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,7 +247,7 @@ Focus is on the specific task.
 *Plugins and libraries for implementing authentication and authorization.*
 
 - [Authenticate plugin](https://github.com/FriendsOfCake/Authenticate) - Authentication classes for AuthComponent.
-- [CakeDC Users plugin](https://github.com/CakeDC/users/tree/3.x) - For allowing users to register and login, manage their profile and account validation. Social Login, RBAC, remember me, SuperUser authorize, forgot password, admin management, etc.
+- [CakeDC Users plugin](https://github.com/CakeDC/users/tree/3.1.x) - For allowing users to register and login, manage their profile and account validation. Social Login with Facebook and Twitter, RBAC, Remember me, SuperUser authorize, forgot password, admin management, etc.
```
Comment (jadb) [inline README.md]: @steinkel, maybe shorten the description - getting too long compared to all the rest.

My suggestion:

```
Complete user management (admin panel, remember me, etc), Facebook/Twitter login, RBAC and more
```

**Comment by dereuromark** [comment]
And you should remove the branch. We always link to the plugin itself and not the specific tree/branch of it. People will be smart enough to find the right branch if you documented it properly.

**Comment by ADmad** [comment]
@steinkel This can be merged if you address the comments made by @jadb and @dereuromark.

**Comment by steinkel** [comment]
thank you guys, updated based on your feedback

File: README.md [hunk 2]
```diff
@@ -247,7 +247,7 @@ Focus is on the specific task.
 *Plugins and libraries for implementing authentication and authorization.*
 
 - [Authenticate plugin](https://github.com/FriendsOfCake/Authenticate) - Authentication classes for AuthComponent.
-- [CakeDC Users plugin](https://github.com/CakeDC/users/tree/3.x) - For allowing users to register and login, manage their profile and account validation. Social Login, RBAC, remember me, SuperUser authorize, forgot password, admin management, etc.
+- [CakeDC Users plugin](https://github.com/CakeDC/users) - Complete user management (admin panel, remember me, etc), Facebook/Twitter login, RBAC, API and more.
```
Comment (dereuromark) [inline README.md]: I think the name is stil wrong. Maybe we can fix that post merge. CakeDC should be removed.

Comment (dereuromark) [inline README.md]: Done in https://github.com/FriendsOfCake/awesome-cakephp/commit/09f7cb4ebc9db0782f6cf6d861ed6e36bf4a8dd2

**Pull request merged**